src/molecules/todoItem/TodoItem.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+//Styles
+import "./TodoItem.css"
+
+const TodoItem = ({todoInfo, onTodoClick }) => {
+
+    const {id, text, isCompleted} = todoInfo;
+
+    const onClick = () => {
```

Comment:
- This is equivalent to calling onTodoClick(id) directly on onClick because we are not doing extra calculation in this method.
- onClick <> onTodoClick